### DISCUSSION TITLE
How do I create a custom CRT linkage option?

### DISCUSSION THREAD

**Discussion by davidmatson** [microsoft/vcpkg#25592] [Q&A]
I see that it's possible to create a custom triplet, including setting VCPKG_CRT_LINKAGE according to the [docs](https://vcpkg.readthedocs.io/en/latest/users/triplets/). But according to that doc, the only valid options are "static" and "dynamic".

If I wanted to use a different CRT linkage approach, such as the "hybrid CRT" approach mentioned here:
https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/HybridCRT.md

Is that feasible to do with vcpkg today? If so, any pointers on where in the code would need to change to add "hybrid" as a new VCPKG_CRT_LINKAGE option?

**Comment by Osyotr** [comment]
It looks like "hybrid CRT" really means "static CRT" with some additional compiler flags (`/nodefaultlib:libucrt.lib` and `/defaultlib:ucrt.lib` with `d` suffix for debug versions).
